tests/zfs-tests/include/libtest.shlib

```diff
@@ -2843,3 +2843,32 @@ function block_device_wait
 		$UDEVADM settle
 	fi
 }
+
+function kill_process 
```

Comment:
There is stray trailing whitespace on a number of lines. Give these things a: sed -i "s|[[:space:]]\+$||" FILE